lib/controllers/characters.js

```diff
@@ -1,2 +1,21 @@
 const { Router } = require('express');
-module.exports = Router();
+const { Character } = require('../models/Character');
+
+module.exports = Router()
+  .get('/', async (req, res) => {
+    const chars = await Character.getAll();
+    const filtered = chars.map(({
+      id,
+      first_name,
+      last_name,
+      quotes
+    }) => ({     
+      id,
+      first_name,
+      last_name,
+      quotes
+    }));
```

Comment:
What is this filtering? It appears that the map is returning each object unchanged.